tgk/test-triage.sh

```
@@ -0,0 +1,38 @@
+
+echo "🧪 Testing Issue Triage MVP"
+echo "==========================="
+
+# Check if GitHub token is available
+if [ -z "$GITHUB_TOKEN" ]; then
+    echo "⚠️  GITHUB_TOKEN not set - running limited tests without API calls"
+    echo ""
+fi
+
+echo "1️⃣ Running unit tests..."
+cd /Users/nolarose/alchmenyrun/tgk
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)